inc/template-tags.php

```diff
@@ -180,8 +184,8 @@ function wp_rig_comments_link() {
 /**
  * Prints edit post/page link when a user with sufficient priveleges is logged in.
  */
-function wp_rig_edit_post_link() {
-	edit_post_link(
+function edit_post_link() {
+	\edit_post_link(
```

Comment:
Yes. `\edit_post_link()` here calls WP core's `edit_post_link()` function (which is in global namespace). Without the slash, it would call `WP_Rig\WP_Rig\edit_post_link()` (because that is the current namespace of the file), which is not what we want, and also would cause an infinite loop.

Reply:
Just checking since it's the only place I spotted this particular markup.

Reply:
Just to clarify, if we were verbose, we would need to add this backslash to every WordPress core function call happening from a namespace.

However, since PHP will fall back to the global namespace automatically unless a function of the same name exists in the current namespace, we're good.